modules/programs/boxxy.nix

```diff
@@ -0,0 +1,110 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.programs.boxxy;
+
+  configPath = "${config.xdg.configHome}/boxxy/boxxy.yaml";
+  settingsFormat = pkgs.formats.yaml { };
+
+  boxxyRulesOpts = types.submodule {
+    freeformType = settingsFormat.type;
+
+    options = {
+      name = mkOption {
+        type = types.str;
+        description = ''
+          Unique identifier of the boxxy rule. This can be any single-line string.
+        '';
+      };
+
+      target = mkOption {
+        type = types.str;
+        default = "";
+        defaultText = "~/.ssh";
```

Comment:
Did you mean to use `example` here? Also, does tilde expansion actually work?

Reply:
Yeah, that was a mistake, fixing that in an another commit.

The tilde expansion works because boxxy launches a shell.